index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+  <link rel="stylesheet" href="style.css">
+  <script src="app.js" defer></script>
+  <title>Tic Tac Toe</title>
+</head>
+<body>
+
+  <div class="container-fluid bg-dark">
+    <div class="container text-light">
+       <div class="row">
+         <div class="col-12 py-4">
+             <h1>Tic Tac Toe </h1>
+             <h2>Game on!</h2>
+         </div>
+         <div class="col-12 text-center">
+           <h3 id="players">Player 1 vs Player 2</h3>
+         </div>
+       </div>
+     </div>
+   </div>
+ </br>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-md-3">
+        <div class="alert alert-warning alert-dismissible fade show" role="alert">
+          <strong>Messages box!</strong>
+          <p id="messages-box">
+
+          </p>
+        </div>
+      </div>
+      <div class="col-12 col-md-6 row">
+        <div class="col-12  d-flex justify-content-center">
+          <div class="boxcontainer">
+            <div class="example" data-id="0">O</div>
+            <div class="example" data-id="1">O</div>
+            <div class="example" data-id="2">O</div>
+            <div class="example" data-id="3">X</div>
+            <div class="example" data-id="4">X</div>
+            <div class="example" data-id="5">X</div>
+            <div class="example" data-id="6">O</div>
+            <div class="example" data-id="7">O</div>
+            <div class="example" data-id="8">O</div>
```

Comment:
These divs should be created without content

Reply:
Thanks Raphael @Oghenebrume50  for taking the time to review our project we change the comment above as you requested.